Cython/Compiler/ExprNodes.py

```diff
@@ -13893,6 +13991,27 @@ def generate_result_code(self, code):
     def analyse_types(self, env):
         return self
 
+
+def coerce_from_soft_complex(arg, dst_type, env):
+    c_api_type = PyrexTypes.CFuncType(
+        PyrexTypes.c_double_type,
+        [ PyrexTypes.CFuncTypeArg("value", PyrexTypes.soft_complex_type, None) ],
+        exception_value = "-1",
+        exception_check = True
+    )
+    call = PythonCapiCallNode(
+        arg.pos,
+        "__Pyx_SoftComplexToDouble",
+        c_api_type,
```

Comment:
This isn't an API, it's just a single function.
```suggestion
    cfunc_type = PyrexTypes.CFuncType(
        PyrexTypes.c_double_type,
        [ PyrexTypes.CFuncTypeArg("value", PyrexTypes.soft_complex_type, None) ],
        exception_value = "-1",
        exception_check = True
    )
    call = PythonCapiCallNode(
        arg.pos,
        "__Pyx_SoftComplexToDouble",
        cfunc_type,
```